.github/workflows/test.yml

```diff
@@ -183,7 +183,8 @@ jobs:
       turbopack_typescript: ${{ steps.ci.outputs.diff != '' || steps.turbopack_typescript.outputs.diff != '' }}
       turborepo_rust: ${{ steps.ci.outputs.diff != '' || steps.turborepo_rust.outputs.diff != '' }}
       turbopack_bench: ${{ steps.ci.outputs.diff != '' || steps.turbopack_bench.outputs.diff != '' }}
-      go: ${{ steps.ci.outputs.diff != '' || steps.turborepo_go.outputs.diff != '' || steps.turborepo_rust.outputs.diff != '' }}
+      go: ${{ steps.ci.outputs.diff != '' || steps.turborepo_go.outputs.diff != ''}}
```

Comment:
`go_lint` and `go_unit` run conditionally based on this output, so we don't need `turborepo_rust` changes to be included

Reply:
I think we need to still run `go_unit` if either `go` or `turborepo_rust` is marked as change since Go depends on Rust code via FFI now and we have unit tests that exercise this.

Reply:
fixed good call